app/[locale]/projects/[projectId]/funding-search/page.tsx

```diff
@@ -74,9 +74,7 @@ const CreateProjectSearchFunder = () => {
   }
 
   async function handleSelectFunder(funder: AffiliationSearch) {
-    const NEXT_URL = routePath('projects.create.projects.search', {
-      projectId: projectId as string,
-    });
+    const NEXT_URL = routePath('projects.create.project.search');
```

Comment:
This page was working for me. I developed it and it does go the to search page after picking a funder.

Why did it need changes?

Also, the routepath seems wrong here. There should be a project ID, because you need to search fo the project, and potentially populate existing data in your project search fields. After which you should associate the found project data with the current project.